appintro/src/main/res/layout-land/appintro_fragment_intro.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
PS: The `-land` layout is really cool :)

Reply:
Thanks :-)